config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
You shoud create database.yml.example and push to git, that has same code as database.yml.
Add database.yml to gitignore and run git rm --cached config/database.yml.